<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificatorContainerWidget</class>
 <widget class="QWidget" name="ModificatorContainerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QTabWidget" name="editorTabWidget">
  <property name="tabPosition">
   <enum>QTabWidget::North</enum>
  </property>
        <widget class="QWidget" name="paramPage">
         <attribute name="title">
          <string>CPACS Editor</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="ModificatorFuselageWidget" name="fuselageModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorTransformationWidget" name="transformationModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorWingWidget" name="wingModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorElementWidget" name="elementModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorSectionsWidget" name="sectionsModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorFuselagesWidget" name="fuselagesModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorWingsWidget" name="wingsModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorPositioningWidget" name="positioningModificator" native="true"/>
                </item>
                <item>
                 <widget class="ModificatorSectionWidget" name="sectionModificator" native="true"/>
                </item>
                <item>
                 <widget class="QWidget" name="applyWidget" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QPushButton" name="cancelButton">
                      <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>0</height>
                        </size>
                      </property>
                      <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>16777215</height>
                        </size>
                      </property>
                      <property name="text">
                        <string>Cancel</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="commitButton">
                      <property name="text">
                        <string>OK</string>
                      </property>
                     </widget>
                    </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="noInterfaceWidget" native="true">
                  <property name="toolTip">
                    <string>Display the interface for the element selected in the tree.</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="toolTip">
                        <string/>
                      </property>
                      <property name="text">
                        <string>Please select an editable element from the CPACS Tree. </string>
                      </property>
                     </widget>
                    </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>80</height>
                    </size>
                  </property>
                 </spacer>
                </item>
              </layout>
             </widget>
            </widget>
          </item>
         </layout>
        </widget>
        <widget class="ModificatorDisplayOptionsWidget" name="displayOptionsWidget" native="true">
         <attribute name="title">
          <string>Display Options</string>
         </attribute>
        </widget>
      </widget>
     </item>
    </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModificatorElementWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorElementWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorFuselageWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorFuselageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorSectionWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorSectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorTransformationWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorTransformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorWingWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorWingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorFuselagesWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorFuselagesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorSectionsWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorSectionsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorWingsWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorWingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorPositioningWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorPositioningWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModificatorDisplayOptionsWidget</class>
   <extends>QWidget</extends>
   <header>modificators/ModificatorDisplayOptionsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
